utils/autoPager.js

```diff
@@ -32,7 +32,7 @@ const {
  */
 async function getIncidentsForRoom(roomId, botId) {
   const incidents = await bdk.getBotData(roomId, botId, 'onCallIncidents');
-  if (!incidents?.body?.[0]?.value) return [];
+  if (incidents?.body?.[0]?.value === undefined) return [];
```

Comment:
are u sure its gonna be undefined and not null ? may wanna soft compare

Reply:
Yeah, this check is just for if incidents.body doesn't exist / body[0] doesn't exist / body[0].value doesn't exist; in all of these cases it will return undefined.
If there is an existing value it makes it down to the code below which does a ```try { JSON.parse }``` which will catch if the value happens to be null etc.